torch_geometric/loader/temporal_dataloader.py

```diff
@@ -17,7 +18,9 @@ class TemporalDataLoader(torch.utils.data.DataLoader):
         **kwargs (optional): Additional arguments of
             :class:`torch.utils.data.DataLoader`.
     """
-    def __init__(self, data: TemporalData, batch_size: int = 1, **kwargs):
+    def __init__(self, data: TemporalData, batch_size: int = 1,
+                 device: Union[torch.device, str] = 'cpu',
```

Comment:
Mh, I feel this implicit device transfer is kinda breaking the usual computation flow of PyTorch. Why do we need this here? Can we drop it?

Reply:
yeah